src/libraries/System.Net.Http/src/System/Net/Http/SocketsHttpHandler/Http2Connection.cs

```diff
@@ -194,7 +194,7 @@ public async ValueTask SetupAsync(CancellationToken cancellationToken)
             try
             {
                 _outgoingBuffer.EnsureAvailableSpace(Http2ConnectionPreface.Length +
-                    FrameHeader.Size + FrameHeader.SettingLength +
+                    FrameHeader.Size + 2 * FrameHeader.SettingLength +
```

Comment:
This calculation was technically wrong before, but 52 vs 58 both end up with at least 64.

Reply:
Nit: consider adding parens around the multiplication. I realize it's not necessary, but it took me a moment to visually parse what was being done.